src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -16,6 +16,21 @@ use std::path::{Path, PathBuf};
 use std::process::exit;
 use std::sync::Arc;
 
+#[derive(Debug)]
```

Comment:
Making the representation of this enum `u8` might allow you to use its values directly as exit codes without casting:
```
#[repr(u8)]
enum ExitCode {...}
```

Reply:
Sadly not (not with #[repr(i32)] either), but good to learn about!